lib/particular-calendars/costa-rica.ts

```diff
@@ -14,5 +15,17 @@ export class CostaRica extends CalendarDef {
       dateDef: { dateFn: 'pentecostSunday', addDay: 4 },
       properCycle: ProperCycles.ProperOfTime,
     },
+
+    our_lady_of_angels: {
+      precedence: Precedences.GeneralSolemnity_3,
+      /*
+       * Ordo Temporis 2023-2024
```

Comment:
I am not against of adding the source title here, however, you didn’t add it in the other place.

Reply:
sure, i'm trying to make a better structure to make our sources clear without having to necessarily go into them so that if the link goes down someday, we know what we were looking at

Reply:
i've added it to the other.